Makefile

```diff
@@ -307,8 +307,18 @@ official-oss-image: ## Build official NGINX OSS with NGINX Agent container image
 	@echo Building image nginx-oss-with-nginx-agent with $(CONTAINER_CLITOOL); \
 	cd scripts/docker/official/nginx-oss-with-nginx-agent/alpine/ \
 	&& $(CONTAINER_BUILDENV) $(CONTAINER_CLITOOL) build -t nginx-oss-with-nginx-agent . \
+	    --build-arg NGINX_AGENT_VERSION=$(shell echo ${VERSION} | tr -d 'v') \
```

Comment:
use a common variable that can be over-written. 

i.e.

```
BUILD_AGENT_VERSION ?= $(shell echo ${VERSION} | tr -d 'v')
```

this allow user input with default. https://www.gnu.org/software/make/manual/html_node/Setting.html